language/llama3-405b/user.conf

```diff
@@ -0,0 +1,13 @@
+# The format of this config file is 'key = value'.
+# The key has the format 'model.scenario.key'. Value is mostly int64_t.
+# Model maybe '*' as wildcard. In that case the value applies to all models.
+# All times are in milli seconds
+#
+*.Offline.min_duration = 600000
+*.Offline.min_query_count = 2000
+
+*.Server.target_qps = 0.5
+*.Server.min_duration = 120000
+*.Server.min_query_count = 100
+
+llama3-405b.Server.sample_concatenate_permutation = 1
```

Comment:
Also need the top level mlperf.conf changes.

Reply:
Updated `mlperf.conf` to have this field and the other configuration for llama3